main.py

```diff
@@ -1,255 +1,111 @@
+    def on_message_callback(self, ch, method, properties, body):
         self.logger.debug("handling delivery...")
-        wes_data_service_messages_handled_total.inc()
+        wes_data_service_messages_total.inc()
 
-        if delivery.pod_uid is None:
-            self.logger.debug("dropping message without pod uid")
-            wes_data_service_messages_invalid_total.inc()
-            delivery.ack()
-            return
+        app_uid = properties.app_id
 
-        pod_state = self.get_or_create_pod_state(delivery.pod_uid)
-
-        if pod_state.pod is None:
-            self.logger.debug("adding delivery %s to backlog for %s", delivery, delivery.pod_uid)
-            wes_data_service_messages_backlogged_total.inc()
-            wes_data_service_messages_in_backlog.inc()
-            pod_state.backlog.append(delivery)
-        else:
-            self.load_and_publish_message(delivery)
-            pod_state.updated_at = self.clock.now()
-
-    def handle_pod(self, pod):
-        self.logger.debug("handling pod event %s (%s)...", pod.name, pod.uid)
-
-        pod_state = self.get_or_create_pod_state(pod.uid)
-        pod_state.pod = pod
-        pod_state.updated_at = self.clock.now()
-
-        self.flush_pod_backlog(pod.uid)
-
-    def get_or_create_pod_state(self, pod_uid):
-        if pod_uid in self.pod_state:
-            return self.pod_state[pod_uid]
-        pod_state = self.new_pod_state()
-        self.pod_state[pod_uid] = pod_state
-        wes_data_service_pods_in_backlog.inc()
-        return pod_state
-
-    def handle_expired_pods(self):
-        # TODO(sean) use pod status (ex. Running) to prolong life instead of blanket timeout
-        self.logger.debug("updating pod state...")
-
-        for pod_uid in list(self.pod_state.keys()):
-            pod_state = self.pod_state[pod_uid]
-
-            if not self.pod_state_expired(pod_state):
-                continue
-
-            self.logger.debug("expiring pod state for %s", pod_uid)
-            for delivery in pod_state.backlog:
-                delivery.ack()
-                wes_data_service_messages_in_backlog.dec()
-                wes_data_service_messages_expired_total.inc()
-            del self.pod_state[pod_uid]
-            wes_data_service_pods_in_backlog.dec()
-            wes_data_service_pods_expired_total.inc()
-
-        self.logger.debug("updated pod state")
-
-    def pod_state_expired(self, pod_state):
-        if pod_state.pod is None:
-            ttl = self.config.pod_without_metadata_state_expire_duration
-        else:
-            ttl = self.config.pod_state_expire_duration
-        return self.clock.now() - pod_state.updated_at > ttl
-
-    def flush_pod_backlog(self, pod_uid):
-        self.logger.debug("flushing pod backlog for %s...", pod_uid)
-        pod_state = self.pod_state[pod_uid]
-        if pod_state.pod is None:
+        if app_uid is None:
+            self.logger.info("reject msg: no pod uid: %r", body)
```

Comment:
Sure. I changed these.

In any case, these are things I generally want to know about during normal run time in case we're accidentally sending broken messages, so I'll always want them. (Basically, I don't think they're debug messages.)